src/components/ide.js

```diff
@@ -57,13 +60,16 @@ const IDE = () => {
       const Firepad = require("firepad")
 
       // initialize Firepad instance under the firepadRef realtimeDB address, and navigate to that address
-      const firepad = Firepad.fromCodeMirror(firepadRef, codeMirror, {richTextToolbar: false,
+      const firepad = Firepad.fromCodeMirror(res, codeMirror, {richTextToolbar: false,
```

Comment:
Codacy found an issue: ['firepad' is assigned a value but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5958117)